docs/attestations/sbom.md

```diff
@@ -68,14 +68,14 @@ enabled via the build arguments.
 For example:
 
 ```dockerfile
-FROM alpine:latest as build
+FROM alpine:latest AS build
```

Comment:
We SHOULD avoid making changes to the files maintained by moby/buildkit unless @EarthBuild requires these changes.

The list of files maintained by us https://github.com/EarthBuild/buildkit/wiki/EarthBuild-files#modified-by-earthbuild

It would be nice not to touch any other files to avoid merge conflicts and ease fork maintenance.